### ISSUE TITLE
Test v0.12.0 release

### ISSUE THREAD

**Issue by amarthadan** [api3dao/airnode#1842]
All NPM packages and Docker images are available under the `0.12.0` tag (or `latest` at the moment).

**Comment by dcroote** [comment]
@bbenligiray would you consider this Done based on your testing?

**Comment by bbenligiray** [comment]
I wouldn't, I only tested those specific issues

**Comment by bbenligiray** [comment]
Do we have a written testing procedure @aTeoke ?

**Comment by dcroote** [comment]
@bbenligiray tested the following v0.12.0 functionality already:
- Reserved parameters are inaccessible in pre-processing and post-processing - [here](https://github.com/api3dao/airnode/issues/1738#issuecomment-1656712765)
- All pre and post processing steps have access to an `endpointParameters` object that includes all parameters sans reserved - [here](https://github.com/api3dao/airnode/issues/1673#issuecomment-1656715227)

I tested the below:

## v0.12.0 testing

tl;dr - everything worked with deployments on AWS and GCP.

I modified `config.json` as follows:

1. Deleted `chains[0].contracts` object in order to test defaulting to API3 deployed AirnodeRrpV0 contract. It worked given the requests below.

2. Deleted `fulfillmentGasLimit` in order to test AirnodeRrpDryRunV0 gas estimation. It worked:

   ```
   [2023-08-16 05:05:11.549] INFO Gas limit is set to 89140 (AirnodeRrp: 38600 + Fulfillment Call: 50540)
   ```

**A checked box indicates success.**

Test that npm packages `admin`, `deployer`, and `validator`, which have a `--help` CLI argument, are compiled correctly using `npx`

- [x] admin
  - [x] `generate-mnemonic`
- [x] deployer (tested more below)
- [x] validator
  - [x] Succeeds with `--config "packages/airnode-validator/test/fixtures/config.valid.json" --secrets "packages/airnode-validator/test/fixtures/secrets.valid.env"`

The following use the `coingecko-http-gateways` integration in `airnode-examples` on Ethereum sepolia.

### Test AWS deployment with `deployer` Docker image:

- Requests
  - [x] Testnet request
  - [x] HTTP gateway request
  - [x] Signed HTTP gateway request
  - [x] Withdrawal request
- Deployer commands (with Docker image)
  - [x] `list`
  - [x] `info`
  - [x] `remove`

### Test GCP deployment with `deployer` Docker image:

As an aside related to #1858, GCP deployment is several fold, possibly approaching an order of magnitude, slower than AWS deployment. I didn't time it for exact values.

- Requests
  - [x] Testnet request
  - [x] HTTP gateway request
  - [x] Signed HTTP gateway request
  - [x] Withdrawal request
- Deployer commands (with Docker image)
  - [x] `list`
  - [x] `info`
  - [x] `remove-with-receipt`

## Failures

None.
